app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -258,18 +258,24 @@ private void updateToolbarButtonStates() {
     }
 
     public String getUrl() {
```

Comment:
Let's add @Nullable here. This might not be expected. I guess we could keep a reference to the url ourselves. We should always have some because we pass one in when we create the Fragment (even if there's no WebView yet).